language/move-lang/rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+edition = "2018"
+use_field_init_shorthand = true
```

Comment:
Should this be in the top-level rustfmt.toml?

Reply:
Ah, I forgot to remove this file. It was just there to turn on experimental features so I could find the spots to fix :) 

Will remove before landing